pkgs/applications/networking/browsers/firefox/packages.nix

```diff
@@ -0,0 +1,107 @@
+{ lib, callPackage, fetchurl, fetchFromGitHub }:
+
+let common = opts: callPackage (import ./common.nix opts); in
```

Comment:
Given the length of both files, I doubt that keeping them together simplifies things, to be honest